x/cork/client/cli/query.go

```diff
@@ -91,26 +92,28 @@ func querySubmittedCorks() *cobra.Command {
 	return cmd
 }
 
-func queryCommitPeriod() *cobra.Command {
+func queryScheduledCorks() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "commit-period",
-		Aliases: []string{"cp"},
+		Use:     "scheduled-corks",
+		Aliases: []string{"scs"},
 		Args:    cobra.NoArgs,
-		Short:   "query commit period data from the chain",
+		Short:   "query scheduled corks from the chain",
 		RunE: func(cmd *cobra.Command, _ []string) error {
 			ctx, err := client.GetClientQueryContext(cmd)
 			if err != nil {
 				return err
 			}
 
 			queryClient := types.NewQueryClient(ctx)
-			req := &types.QueryCommitPeriodRequest{}
+			req := &types.QueryScheduledCorksRequest{}
 
-			res, err := queryClient.QueryCommitPeriod(cmd.Context(), req)
+			res, err := queryClient.QueryScheduledCorks(cmd.Context(), req)
 			if err != nil {
 				return err
 			}
 
+			writer := os.Stdout
+			writer.Write([]byte(res.String()))
```

Comment:
Why is this here?

Reply:
I think this is an artifact of debugging that I forgot to remove.